src/index.d.ts

```diff
@@ -86,6 +262,21 @@ declare namespace admin.app {
     messaging(): admin.messaging.Messaging;
     projectManagement(): admin.projectManagement.ProjectManagement;
     storage(): admin.storage.Storage;
+    /**
+     * Renders this app unusable and frees the resources of all associated
+     * services.
```

Comment:
Couple of developers have pointed out that we should explain this method a bit more in our documentation (see #489 for example). Can we add a line here indicating that when running the SDK locally, this method must be called to ensure graceful termination of the process?